server/views/pages/applications/list/index.njk

```diff
@@ -1,7 +1,7 @@
-{%- from "moj/components/sub-navigation/macro.njk" import mojSubNavigation -%}
-{%- from "moj/components/filter/macro.njk" import mojFilter -%}
-{%- from "govuk/components/table/macro.njk" import govukTable -%}
-{%- from "govuk/components/tabs/macro.njk" import govukTabs -%}
+{% from "moj/components/sub-navigation/macro.njk" import mojSubNavigation %}
+{% from "moj/components/filter/macro.njk" import mojFilter %}
```

Comment:
Not sure if you want to remove these? It gets rid of lots of whitespace in the output I think?